### ISSUE TITLE
Email alerts module not working properly - impossible to save e-mail address

### ISSUE THREAD

**Issue by Tomatjul** [PrestaShop/PrestaShop#33389]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

I just upgraded from 1.7.8.2 to 8.1.0 and since then, email alerts are not working anymore.
Even after uninstalling/installing, I still have the same error message when trying to save the e-mail address.
Error message: **Please enter one (or more) email address for the new order notification**

I'm obviously not the only one as I noticed here that someone seems to have the same issue but there is no answer :
https://www.prestashop.com/forums/topic/1076029-mail-alerts-problem/

### Expected behavior

I would like to receive e-mail alerts on new orders.
I would like to save an e-mail address in the " ps_emailalerts / Merchant notifications / New Order / Send to : " module field.

### Steps to reproduce

1. go to BO
2. Menu : Modules : Module Manager
3. search and configure ps_emailalerts
4. Switch ON : "New Order" in "Merchant notifications" block
5. Insert an e-mail address in the "Send to: " field
6. Press Save
7. Error message displayed : **Please enter one (or more) email address for the new order notification**

### PrestaShop version(s) where the bug happened

8.1.0

### PHP version(s) where the bug happened

8.1

### If your bug is related to a module, specify its name and its version

pe_emailalerts

**Comment by Tomatjul** [comment]
I must mention that what I have uninstalled/installed is the ps_emailalerts module version 2.4.1.
thanks

**Comment by kpodemski** [comment]
As you can now type multiple emails in the configuration field, you have to type the email address and click the "Enter" key first.

Please try that first.

**Comment by kpodemski** [comment]
If that helps, we'll try to improve the helper text next to the field for the next version. Let us know :)

**Comment by Tomatjul** [comment]
Czecs Krystian,
Well, thanks a lot. I could insert the e-mail address as you mentioned. Obviously, I did not pay enough attention to that text. And now I'm waiting for an order to check out alerts are working.

However before I reinstalled and reset the configuration, the e-mail address was correct and the configuration value MA_MERCHANT_ORDER_EMAILS was also correct. But alerts weren't working. So still I let you know the result in a few hours.
